tests/scalability/start_ec2_medic.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+#Requires jq
+
+#Base 64 encode the user-data script to start medic-os
+jq '.UserData = "'$(base64 medic-os.sh -w 0)'"' launch-specification.json >> launch-specification-medic-os.json
+
+echo Starting Instance Requests
+SpotInstanceRequestId=$(aws ec2 request-spot-instances --spot-price '0.333' --instance-count 1 --type 'one-time' --launch-specification file://launch-specification-medic-os.json --block-duration-minutes 60 | jq .SpotInstanceRequests[0].SpotInstanceRequestId -r)      
+echo Getting Instance ID
+instanceID=$(aws ec2 describe-spot-instance-requests --spot-instance-request-ids $SpotInstanceRequestId | jq .SpotInstanceRequests[0].InstanceId -r )
+while [ "$instanceID" = null ]
+do
+sleep 5
+instanceID=$(aws ec2 describe-spot-instance-requests --spot-instance-request-ids $SpotInstanceRequestId | jq .SpotInstanceRequests[0].InstanceId -r )
+echo "Sleeping while waiting for instance ID to be setup."
+echo Instance id is $instanceID
+done 
+echo Getting PublicDnsName
+PublicDnsName=$(aws ec2 describe-instances --instance-ids $instanceID | jq .Reservations[0].Instances[0].PublicDnsName -r)
+echo DNS is $PublicDnsName
+
+if [ -z "$PublicDnsName" ]
+then
+echo dns name not setting. Trying to get again
+PublicDnsName=$(aws ec2 describe-instances --instance-ids $instanceID | jq .Reservations[0].Instances[0].PublicDnsName -r)
+fi
+
+if [ -z "$PublicDnsName" ]
+then
+echo Did not get public dns name. Exiting now.
+exit 1
+fi
+
+url=https://$PublicDnsName
+
+echo "::set-env name=MEDIC_URL::$url"
+
+echo Begin Checking $url/api/info is up
+version=$(curl -s $url/api/info -k  | jq .version -r)
+
+until [ "$version" = 0.1.0 ]
+do
+version=$(curl -s $url/api/info -k  | jq .version -r)
+sleep 10
+echo Sleeping again. Version is $version
+echo
+echo 
+done
+
+echo Api Is up
+
+MEDIC_CONF_URL=https://medic:medicScalability@$PublicDnsName
+
+
+cp -r ./csv ../../config/standard/
+
+cd ../../config/standard/
+
+echo installing pip
+sudo apt-get -q install python-pip -y
+
+echo installing pyxform
+sudo python -m pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic -q
+
+echo installing medic-conf
+npm install https://github.com/medic/medic-conf.git#307_force_arg
+
+# echo Uploading settings and seeding data
+echo medic-conf url is $MEDIC_CONF_URL
+$(npm bin)/medic-conf --url="$MEDIC_CONF_URL" --force --accept-self-signed-certs upload-app-settings \
+    convert-app-forms \
+    convert-collect-forms \
+    convert-contact-forms \
+    delete-all-forms \
+    upload-app-forms \
+    upload-collect-forms \
+    upload-contact-forms \
+    upload-resources \
+    upload-custom-translations  \
+    csv-to-docs \
+    upload-docs \
+    create-users \
```

Comment:
Yeah there are no attachments.  I'd think that'd be something we'd want to check as it affects replication time.

Reply:
I wrote this script a while ago, I gave it a turn and it seems like it still works. 

https://github.com/dianabarsan/generate-form-attachments/blob/master/src/index.js
You just need to export a COUCH_URL and run it. It will create attachments for every report in the database (albeit not correctly most likely).

Reply:
I think that would be fine. My quick read/understanding makes it seem like the attachment should be the same each time. Is that correct?

Reply:
Same for the same form, yea. I basically transpiles the `fields` property into an XML and wraps it in what enketo expects.

Reply:
If you're willing to use this, I can perfect it, unit test it and everything :)

Reply:
Yeah if you want to add it to the never ending list that'd be great.

Reply:
@dianabarsan How did you get on with this? Should we create a new issue to add attachments after this is merged?

Reply:
I wrote a script 2 weeks ago and passed it to @newtewt . 
https://github.com/medic/support-scripts/tree/master/generate-form-attachments

@newtewt did you give it a shot? Would you like me to integrate it ?

Reply:
Yeah I had gotten time to review how it works yesterday. I'll add it today.